packages/crew_profile.rb

```diff
@@ -0,0 +1,47 @@
+require 'package'
+
+class Crew_profile < Package
+  description 'A profile for Chromebrew, to be used instead of ~/.*rc'
+  homepage 'https://github.com/skycocker/chromebrew'
+  version '0.4.3'
+  source_url 'https://github.com/skycocker/chromebrew/raw/511ee9cb3138e113df74ddf0dee057ef5d9a46fd/README.md'
+  source_sha256 '27b201cec82d903a1856972e6d7ff1ac58a67c761d729ecd7fd14f24fa9d9901'
+
+  depends_on "xdg_base"
+
+  def self.install
+    system "mkdir", "-p", "#{CREW_DEST_PREFIX}/etc"
+    system "touch", "#{CREW_DEST_HOME}/.profile"
+    system "ln", "-s", "#{CREW_DEST_HOME}/.profile", "#{CREW_PREFIX}/etc/profile"
```

Comment:
Does Chrome OS even use this on login?  I know it uses `~/.bash_profile` since the default shell is bash.

Reply:
No. Read the postinstall.